File: Easy/TwoSum.cc
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> output;
        int diff;
        for (int i = 0; i < nums.size(); ++i) {
            diff = target - nums[i];
            for (int j = i + 1; j < nums.size(); ++j) {
                if (diff == nums[j]) {
                    output.push_back(i);
                    output.push_back(j);
                }
            }
        }
        return output;
    }
};

File: Easy/TwoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let diff = 0;
    const output = new Array(2);
    for (let i = 0; i < nums.length; i++) {
        diff = target - nums[i];
        output[0] = i;
        for (let j = i + 1; j < nums.length; j++) {
            if (nums[j] == diff) {
                output[1] = j;
                return output;
            }
        }
    }
};
